_applications/tensorflow_mnist.adoc

```diff
@@ -0,0 +1,76 @@
+= Tensorflow MNIST
+:page-link: tensorflow_mnist
+:page-weight: 100
+:page-labels: [Tensorflow, S2I]
+:page-layout: application
+:page-menu_template: menu_tutorial_application.html
+:page-description: This demo shows how to use source-to-image Tensorflow Serving build to deploy a tensorflow serving prediction endpoint on Openshift. The s2i build provides a GRPC microservice endpoint for web applications to send queries to be evaluated against the tensorflow model.
+:page-project_links: ["https://github.com/radanalyticsio/tensorflow-serving-s2i"]
+
+[[introduction]]
+== Introduction
+
+MNIST digit identification application provides an example of how to use s2i build to deploy a tensorflow prediction endpoint on Openshift.
+Input data is taken from  http://yann.lecun.com/exdb/mnist/[MNIST site].
+
+
+[[architecture]]
+== Architecture
+
+The system is composed of:
```

Comment:
are these all in a single pod, multiple pods, etc.

i think we should say something like:

"The system is composed of 3 pods:"

but make it fit the deploy, if it's all 1 pod that's fine, i just think we should state that.